### PR TITLE
Add net input to telegraf conf template

### PR DESCRIPTION
**Pull request by abednarik**
Net Input is part of the system plugin. Can we have this option to enabled/disable this one?

Great role,
Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,8 @@ Role Variables
...
 | telegraf_input_swap_enabled                   | true                           | Enable swap input                                                                                                                                                                                                                                                                      |
 | telegraf_input_system_enabled                 | true                           | Enable system input                                                                                                                                                                                                                                                                    |
+| telegraf_input_net_enabled                    | true                           | Enable network input                                                                                                                                                                                                                                                                    |
```
Comment (kbrebanov) [inline README.md]: The default should be set to 'false' so that currently provisioned Telegraf instances don't change.

File: defaults/main.yml [hunk 2]
```diff
@@ -67,3 +67,5 @@ telegraf_input_processes_enabled: true
 telegraf_input_swap_enabled: true
 
 telegraf_input_system_enabled: true
+
+telegraf_input_net_enabled: true
```
Comment (kbrebanov) [inline defaults/main.yml]: The default should be set to 'false' so that currently provisioned Telegraf instances don't change.

**Review by kbrebanov — CHANGES_REQUESTED** [review]
Default value for **telegraf_input_net_enabled** should be set to **false**

**Comment by abednarik** [comment]
Sure @kbrebanov. PR updated.

Cheers.

**Review by kbrebanov — APPROVED** [review]

**Comment by abednarik** [comment]
Thanks

**Pull request merged**